### PR TITLE
Add two preCICE theses.

### LINKED ISSUE #106: Add another two
From the mailing list:

```
@mastersthesis{2018,
  author    = {Schneider, David},
  title     = {Simulation von Fluid-Struktur-Interaktion mit der Kopplungsbibliothek preCICE},
  school    = {Universität Siegen},
  year      = {2018},
  note      = {Bachelorarbeit}
}
```

```
@mastersthesis{2017,
  author    = {Rave, Kevin},
  title     = {Kopplung von OpenFOAM und deal.II Gleichungslösern mit preCICE zur Simulation multiphysikalischer Probleme},
  school    = {Universität Siegen},
  year      = {2017}
}
```

### PR DESCRIPTION
**Pull request by bangerth**
Fixed #106.

### CODE REVIEW

**Comment by drwells** [comment]
We use `\"{o}` for umlauts in other entries. I think we should convert these to that style.

**Comment by drwells** [comment]
IIRC we still have problems with unicode here. If I am wrong then this is good as-is.

**Comment by bangerth** [comment]
I've fixed the two umlauts.

File: publications-2017.bib [hunk 1]
```diff
@@ -1280,5 +1280,11 @@ @PhdThesis{RyanRGrovePhD2017
   Url                      = {https://tigerprints.clemson.edu/all_dissertations/1985/}
 }
 
+@mastersthesis{rave2017,
+  author    = {Rave, Kevin},
+  title     = {Kopplung von OpenFOAM und deal.II Gleichungslösern mit preCICE zur Simulation multiphysikalischer Probleme},
```
Comment (drwells) [inline publications-2017.bib]:
```suggestion
  title     = {Kopplung von OpenFOAM und deal.II Gleichungsl{\"o}sern mit preCICE zur Simulation multiphysikalischer Probleme},
```

**Comment by bangerth** [comment]
Ah, that's where the `\"o` was -- I had only seen all of the `\"a`s. Thanks for pointing it out -- fixed now!

File: publications-2018.bib [hunk 2]
```diff
@@ -922,4 +922,12 @@ @ARTICLE{Freddi2018573
   pages={573-592}
 }
 
+@mastersthesis{schneider2018,
+  author    = {Schneider, David},
+  title     = {Simulation von Fluid-Struktur-Interaktion mit der Kopplungsbibliothek preCICE},
```
Comment (drwells) [inline publications-2018.bib]: I am sorry that I keep finding stuff, but we should at least have `{preCICE}` here to get the capitalization correct, right? I think we are more lax on other capitalization issues.

Comment (bangerth) [reply publications-2018.bib]: I mulled this at the other entry I changed, but then realized that it probably doesn't matter. The issue is that bibtex generally converts everything into lower-case for the titles of articles, and consequently if one wants something capitalized, it needs to be enclosed in `{...}`. 

But bibtex doesn't do that for book titles, which are copied verbatim to latex. I suspect that that's also the case for thesis titles, and that consequently the changes I made in the other entry are pointless (though also harmless).

Let's go with this version and I'll look at how it comes out. If it's wrong, I'll fix it up in a follow-up.

Comment (drwells) [reply publications-2018.bib]: Sounds good.

Comment (bangerth) [reply publications-2018.bib]: It looks correct on the website!

**Pull request merged**